### PR TITLE
Contributing Guide in `.rst`

### PR DESCRIPTION
**Pull request by arjxn-py**
Includes Contributing Guide in `readthedocs` while also having a converted version of `CONTRIBUTING.md` in `.rst`

### CODE REVIEW

**Comment by arjxn-py** [comment]
I think it should be nice to [enable `readthedocs` build on PRs](https://docs.readthedocs.io/en/stable/pull-requests.html)?

File: docs/source/contributing.rst [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+.. code-block:: bash
+
+    pip uninstall jupytercad
+
+In development mode, you will also need to remove the symlink created by ``jupyter labextension develop`` command. To find its location, you can run ``jupyter labextension list`` to figure out where the ``labextensions`` folder is located. Then you can remove the symlink named ``jupytercad`` within that folder.
+
+Packaging the extension
+-----------------------
+
+See `RELEASE <RELEASE.md>`_
```
Comment (arjxn-py) [inline docs/source/contributing.rst:100]: Should I remove this from `contributing.rst` page or I can quickly make the `release.rst`?

**Comment by arjxn-py** [comment]
I also propose to take some inspiration from [JupyterLite Contribution Guide](https://jupyterlite.readthedocs.io/en/stable/contributing.html), i'll be happy to make some changes accordingly if permitted.

File: docs/source/contributing.rst (hunk 1 above)
Comment (trungleduc) [reply docs/source/contributing.rst:100]: You can remove this section. Having the release guide on Github is enough.

Comment (arjxn-py) [reply docs/source/contributing.rst]: Done in [`fc660f7`](https://github.com/jupytercad/JupyterCAD/pull/381/commits/fc660f75b1017fde98c66f431b1c1414d4b90912)

**Comment by arjxn-py** [comment]
Have organized media files in `assets/` folder [`a7ee733`](https://github.com/jupytercad/JupyterCAD/pull/381/commits/a7ee733901787175406ea8815ff9bdc0f29a47ad), hoping that's alright.

**Review by trungleduc — APPROVED** [review]

**Comment by trungleduc** [comment]
Thanks!

**Pull request merged**